Gopkg.lock

```diff
@@ -1,6 +1,11 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
+  name = "github.com/BurntSushi/toml"
+  packages = ["."]
+  revision = "b26d9c308763d68093482582cea63d69be07a0f0"
```

Comment:
Any chance we could move this commit, maybe to the head commit. This is tied to the 0.3.0 release I believe, but I'd like to pick up [this commit](https://github.com/BurntSushi/toml/commit/4678bf20d0945128c23c31e748cde56476e8fd91) In particular to get the updated license for that package :-)

Reply:
done, thanks